docker-compose.yml

```diff
@@ -149,7 +149,14 @@ services:
       disable: true
 
   superset-node:
-    image: node:20
+    build:
```

Comment:
bycatch - somehow superset-node wasn't using the proper target, which explains some issues with `zstd` we've seen before, which gets installed at the os-level here. There's also the "memory snag" script and the DEV_MODE logic to skip the actual webpack build since we run `npm run dev` on startup.

Reply:
I thought I had caught/fixed that in some other PR, but guessing it got lost in the shuffle.

Reply:
Nice, thanks for fixing that